packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/paymentMethodCollection.js

```diff
@@ -0,0 +1,5 @@
+/**
+ * This file is augmented at build time using the @magento/venia-ui build
+ * target "payments", which allows third-party modules to add new payment component mappings.
+ */
+export default {};
```

Comment:
I would add to the docs the potential type of the default object.

```
{
   [paymentCode]: paymentCodeComponent
}
```

Or the post build representation of this file:

```es6
import braintree from '@magento/venia-ui/lib/components/CheckoutPage/paymentMethods/braintree`;
import paypal from '@third_party_package/path/to/the/component';

export default {
   braintree: braintree,
   paypal: paypal
}
```